geometry_msgs/PoseStamped start
geometry_msgs/PoseStamped final
string planner
---
int64 result
geometry_msgs/PoseArray path
